core/src/main/java/org/elasticsearch/common/network/NetworkModule.java

```diff
@@ -236,9 +235,10 @@ private CompositeTransportInterceptor(List<TransportInterceptor> transportInterc
 
         @Override
         public <T extends TransportRequest> TransportRequestHandler<T> interceptHandler(String action, String executor,
-                                                                                        TransportRequestHandler<T> actualHandler) {
+                                                                                        TransportRequestHandler<T> actualHandler,
+                                                                                        boolean forceExecution) {
```

Comment:
just a nit: maybe pass forceExecution before actualHandler so that the order is similar to the `registerRequestHandler` method